src/math/trig.c

```diff
@@ -100,6 +159,7 @@ void cordic_approx(int32_t th_rad_fxp, int32_t a_idx, int32_t *sign, int32_t *b_
 	/* Q2.30 format -sine, cosine*/
 	*th_cdc_fxp = th_rad_fxp;
 }
+#endif /* Generic C version */
```

Comment:
The comment should match the if i.e. CORDIC_HIFI3 so its easy to match when viewing the brackets